install_ant-media-server.sh

```diff
@@ -256,6 +257,22 @@ fi
 unzip $ANT_MEDIA_SERVER_ZIP_FILE
 check
 
+VER=`unzip -p ant-media-server/ant-media-server.jar META-INF/MANIFEST.MF | grep "Implementation-Version"|cut -d' ' -f2 | tr -d '\r'`
+
+if [[ $VER == 2.4* || $VER == 2.3* || $VER == 2.2* ]]; then
+  if [ "$ID" == "ubuntu" ]; then
+    $SUDO apt-get update -y
+    $SUDO apt-get install openjdk-11-jdk -y
+    check
+  fi
+else
+  if [ "$ID" == "ubuntu" ]; then
```

Comment:
For the version 2.1 and below, it install java-11-jre however it should install  java8
Check the comment at line 269